AbletonMCP_Remote_Script/__init__.py

```
@@ -800,6 +1260,366 @@ def _find_browser_item_by_uri(self, browser_or_item, uri, max_depth=10, current_
             return None
     
     # Helper methods
+
+    def _safe_getattr(self, obj, attr, default=None):
+        """Read a Live API attribute that may raise when unavailable."""
+        try:
+            if hasattr(obj, attr):
+                return getattr(obj, attr)
+        except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)